src/internal.d.ts

```diff
@@ -115,13 +113,6 @@ export interface VNode<P = {}> extends preact.VNode<P> {
 	_vnodeId: number;
 }
 
-export type InternalTypeFlags =
-	| CONSTANTS['TYPE_TEXT']
-	| CONSTANTS['TYPE_ELEMENT']
-	| CONSTANTS['TYPE_CLASS']
-	| CONSTANTS['TYPE_FUNCTION']
-	| CONSTANTS['TYPE_COMPONENT'];
-
```

Comment:
I think the change of the TYPE_* flags from number constants (e.g. `4`) to bit shifts (`1 << 2`) made these types just `number` instead of constants :/

I still prefer the `1 << 2` syntax though cuz it is easier to see how many flags we have and what the next number is if we need to add or modify a flag. So I went ahead and removed these types since it doesn't provide us any extra typing info or guarantees.